cli/utils/ts-linter.js

```diff
@@ -22,18 +22,16 @@ function lintSync() {
 
   // Convert buffers to strings.
   let output = [];
-  if (spawnResult.output) {
-    spawnResult.output.forEach((buffer) => {
-      if (buffer === null) {
-        return;
-      }
-
-      const str = buffer.toString().trim();
-      if (str) {
-        output.push(str);
-      }
-    });
-  }
+  spawnResult.output.forEach((buffer) => {
```

Comment:
For clarification, this was incorrectly added in a previous PR due to a misconfigured unit test.